### PR TITLE
Access to raw logs

### PR DESCRIPTION
**Pull request by dweebo**
As described in #391, provide access to raw device logs in text format.

I just added a link to the logs from the html log page.

### CODE REVIEW

**Comment by edenman** [comment]
I think we'll want to also do this part: `This can probably be off by default and enabled via a gradle flag.`

**Comment by dweebo** [comment]
Ok.  What do you think about having the config be a flag that is either html logs or raw logs?  I envision users wanting either or but probably not both.  I personally would use the raw logs because scrolling is awkward with the html logs.

The default would be html logs

**Comment by JakeWharton** [comment]
I don't like the idea of a switch. I'd rather have it there all the time if people want it.

File: spoon-runner/src/main/resources/page/log.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
             <div class="hero-unit">
                 <h1>{{title}}</h1>
                 <p>{{subtitle}}</p>
+                <a href="{{rawFile}}">Raw log file</a>
```
Comment (JakeWharton) [inline spoon-runner/src/main/resources/page/log.html]: Do you have a screenshot of what this looks like? I'm not sure this is the right place for the link.

**Comment by dweebo** [comment]
screen shot 2017-10-13 at 10 35 43 am

File: spoon-runner/src/main/resources/page/log.html (hunk 1 above)
Comment (dweebo) [reply spoon-runner/src/main/resources/page/log.html]: yeah i wasn't super happy with it.  maybe put a different link on the main page next to the html log link?

**Comment by dweebo** [comment]
Maybe instead like this?

screen shot 2017-10-13 at 10 59 22 am

**Comment by edenman** [comment]
I like the idea of putting it over on the right.  And I'm ok with not having a switch.

**Comment by edenman** [comment]
Thanks @dweebo!

**Pull request merged**